lib/rpc/handler.ex

```diff
@@ -0,0 +1,22 @@
+defmodule UltraDark.RPC.Handler do
+  use JSONRPC2.Server.Handler
+  alias UltraDark.RPC.{Client, Peers}
+
+  @doc """
+    Accept incoming connection from a node.
+    Adds it to Peers.nodes for future broadcasting
+  """
+  def handle_request("add_node", [host, port]) do
```

Comment:
Is there any way to change this `"add_node"` from a string to an atom? `:add_node` both looks cleaner and is easier to pattern match on

Reply:
atom doesn't work because the "type" is lost since it's TCP connection

Reply:
Gotcha, fair enough! Thanks for the clarification